packages/lib/src/vis/tiles/hooks.ts

```diff
@@ -14,12 +14,17 @@ export function useScaledVisibleDomains(size: Size): {
   const { abscissaConfig, ordinateConfig } = useAxisSystemContext();
 
   const xScale = scaleLinear({
-    domain: abscissaConfig.visDomain,
+    domain: abscissaConfig.flip
+      ? abscissaConfig.visDomain.reverse()
+      : abscissaConfig.visDomain,
```

Comment:
I hesitated to use `dataToWorld` (which handles the flip) and add a "worldToArray" scale then instead of taking care myself of the flip... but it doesn't sounds simpler.

Reply:
yes, it should be equivalent to `getCanvasScale(config, canvasSize) +/- canvasSize/2`

Reply:
Here is a way to share code with `getCanvasScale`: https://github.com/silx-kit/h5web/pull/1052/commits/bc1d1a526d6ba0fe49cafeb11dc40192c3ce9a0c
What do you think?

Reply:
Looks good ! I would add `clamp` as an optional parameter to `getDomainScale` to solve https://github.com/silx-kit/h5web/pull/1052#discussion_r841662636

Reply:
I reworked this part because there was an issue with the flip (not needed here since it is handled at the top in a group).
It now calls `createAxisScale` directly, rather than sharing code with `getCanvasScale`